pinc/ImageUtils.inc

```diff
@@ -5,15 +5,27 @@ use Symfony\Component\Process\Process;
 class ImageUtils
 {
     public const IMAGE_OK = 1;
-    public const IMAGE_CORRUPT = 2;
-    public const IMAGE_SKIPPED = 3;
+    public const IMAGE_WARNING = 2;
+    public const IMAGE_CORRUPT = 3;
+    public const IMAGE_SKIPPED = 4;
 
     /** @var array<string, bool> */
     protected array $external_tools = [
         "pngcheck" => false,
         "jpeginfo" => false,
     ];
 
+    // These strings are returned from the validation tools but don't
+    // prevent the image from being loaded/used. Let's treat these substrings
+    // as warnings, not errors.
+    /** @var string[] */
+    public array $warning_strings = [
+        "illegal (unless recently approved) unknown, public chunk eXIf",
+        "invalid gAMA value (0.0000)",
+        "invalid tIME year (1969)",
+        "tEXt text contains NULL character(s)",
+    ];
```

Comment:
I pulled an image for each of the last two for one of my test loads. They were identified as errors, not warnings, and wouldn't load.

Reply:
Ah! Yes, I had the `stripos()` params backwards. Good catch!